content/articles/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/index.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/
+title: How to use the symbolic maths toolbox in MATLAB to analyze the Fourier series
+description: In this article, we are going to use the Symbolic math toolbox in MATLAB to analyze the Fourier series
+author: linet-achieng
+date: 2022-01-12T00:00:00-11:10
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/hero.jpg
+    alt: How to use the symbolic math toolbox in MATLAB to analyze the Fourier series hero image
+---
+
+### How to use the Symbolic math toolbox in MATLAB to analyze the Fourier series
+
+### Introduction
+Symbolic math toolbox provides an easy, intuitive and complete environment to interactively learn and apply math operations such as calculus, algebra, and differential equations. It can also perform common analytical computation such as differentiation and integration to get close form results, simplify and manipulate expression for great insights and solve algebraic and differential equations.
+
```

Comment:
```suggestion
Symbolic math toolbox provides an easy, intuitive and complete environment to interactively learn and apply math operations such as calculus, algebra, and differential equations. It can also perform common analytical computation such as differentiation and integration to get close form results, simplify and manipulate expression for great insights and solve algebraic and differential equations.
<!--more-->
```

Reply:
@linetachieng, kindly add that more tag as suggested by @srishilesh. @srishilesh, what is it for though? I've never encountered it.

Reply:
This comes as a part of the front matter, which is to be added by the peer reviewer.

It separates the first paragraph with the rest of the article.

So, the first paragraph will be displayed as a description along with the title and hero image.

For more info, refer the peer reviewer guidelines for adding front matter.

Reply:
> This comes as a part of the front matter, which is to be added by the peer reviewer.
> 
> It separates the first paragraph with the rest of the article.
> 
> So, the first paragraph will be displayed as a description along with the title and hero image.
> 
> For more info, refer the peer reviewer guidelines for adding front matter.

Alright